deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -347,28 +347,26 @@ extension WebxdcViewController: WKScriptMessageHandler {
 extension WebxdcViewController: WKURLSchemeHandler {
     func webView(_ webView: WKWebView, start urlSchemeTask: WKURLSchemeTask) {
         if let url = urlSchemeTask.request.url, let scheme = url.scheme, scheme == INTERNALSCHEMA {
+            let data: Data
+            let mimeType: String
+            let statusCode: Int
             if url.path == "/webxdc-update.json" || url.path == "webxdc-update.json" {
                 let lastKnownSerial = Int(url.query ?? "0") ?? 0
-                let data = Data(
+                data = Data(
                     dcContext.getWebxdcStatusUpdates(msgId: messageId, lastKnownSerial: lastKnownSerial).utf8)
-                let response = URLResponse(url: url, mimeType: "application/json", expectedContentLength: data.count, textEncodingName: "utf-8")
```

Comment:
explicit setting of textEncoding could be done with the changed constructor below, however, the default for "application/json" is utf-8, and webkit seem to respect that.